coverage_config_aarch64.json.sample

```diff
@@ -1,5 +1,5 @@
 {
   "coverage_score": 90,
-  "exclude_path": "my_autogenerated_file.rs,second_file.rs",
+  "exclude_path": "my_autogenerated_file.rs|second_file.rs",
```

Comment:
I think technically the fullstops here should be escaped

Reply:
Do you mean for json parsing?

Reply:
If I use `virtio-bindings|mock\.rs` I need to escape `\` so it should be `virtio-bindings|mock\\.rs` , but I didn't see any issue using just `.`

Reply:
Because `.` in regex matches anything, so this would match files named `second_file[any character]rs`. Although maybe not the biggest problem, give that such files are highly unlikely to actually contain code

Reply:
@roypat oh, I was forgetting about the regular expression, yes you are right! I'll update ;-)